client/context/context.go

```diff
@@ -249,9 +240,44 @@ func (ctx CLIContext) WithBroadcastMode(mode string) CLIContext {
 	return ctx
 }
 
+// Print outputs toPrint to the ctx.Output based on ctx.OutputFormat which is
+// either text or json. If text, toPrint will be YAML encoded. Otherwise, toPrint
+// will be JSON encoded using ctx.Marshaler. An error is returned upon failure.
+func (ctx CLIContext) Print(toPrint interface{}) error {
```

Comment:
Since this appends a new line the output, maybe its name should be `Println`?

```suggestion
func (ctx CLIContext) Println(toPrint interface{}) error {
```

Reply:
Sure